recipes/zyre/all/Findzyre.cmake

```diff
@@ -0,0 +1,52 @@
+if(CONAN_INCLUDE_DIRS_ZYRE AND CONAN_LIBS_ZYRE)
```

Comment:
This file can be removed.
conan uses generators such as [cmake_find_package](https://docs.conan.io/en/latest/reference/generators/cmake_find_package.html) generators to provide `FindXXX.cmake` scripts.
Also note that your custom `FindXXX.cmake` script won't co-operate with the `cmake_find_package_multi` generator.